Admin Screen Implementation

tremainebuchanan opened the pull request:
This pull request resolves issues #59, #58, #78, and #69.

tremainebuchanan committed:
Created admin page

tremainebuchanan committed:
Three of four admin features completed

tremainebuchanan committed:
Fixed bug in rendering dashboard

tremainebuchanan committed:
Comments placed in critical areas. Admin cards' width extended to accomodate
addition of new column.

tremainebuchanan committed:
Merge branch 'supply-input-impl' into admin-impl
Changes done on endpoint in supply-input-impl added.

tremainebuchanan committed:
Application root 404 error now fixed [issue 70]

tremainebuchanan committed:
closes #70

tremainebuchanan committed:
User workflow being implemented.
The time-to-live feature of mongo deletes all documents from the collection
on which the time-to-live is set. Therefore, it would be necessasry to
create a new model to create new users that expire after a predetermined
time.

tremainebuchanan committed:
Field added to User and New User model

tremainebuchanan committed:
Merge branch 'develop' into admin-impl

Conflicts:
	models/db.js
	public/javascripts/controllers/dashboard-ctrl.js

tremainebuchanan committed:
Resolved merge issues.

tremainebuchanan committed:
Added export line to db.js file for NewUser model

tremainebuchanan committed:
Email support now incorporated

tremainebuchanan committed:
changing logic for user creation

matjames007 commented:
@tremainebuchanan I realized that you are trying to compensate for the inability to use the TTL method... but in compensating I think we might be making it a little more complicated.  The suggestion was made on slack to do the registration then admin approval.  But I am guessing this wasn't adopted, hence my disconnect.

tremainebuchanan committed:
User creation workflow completed.
The user can now navigate to a /register route and create
an account. The admin will see such request and approve
the user. Validation of other fields in the form will be
done in another issue.

tremainebuchanan commented:
Closing pull request. Bug occurred with another branch where commits after the date of the pull request was not included in merge.
